.github/workflows/continuous-integration.yml

```diff
@@ -41,4 +41,6 @@ jobs:
 
       - name: Bazel Test All (opt)
         run: |
-          bazel test -c opt --noshow_progress --test_output=errors -- //xls/...
+          # Explicitly build all targets in `zstd` package without running the tests
+          bazel build -c opt --noshow_progress --test_output=errors //xls/modules/zstd:all
```

Comment:
can we build `//xls/...` instead?

Reply:
changed to
```
bazel build -c opt --noshow_progress --test_output=errors -- //xls/... -//xls/modules/zstd/benchmarks/...
```
so that the main CI workflow won't run the IR benchmarks and time consuming place and route rules